### DISCUSSION TITLE
如何在unpkg.com引用waline v1版本？

### DISCUSSION THREAD

**Discussion by elkan1788** [walinejs/waline#1042] [Q&A] [answered]
近期发现 `jsdelivr.net` 的服务不是很稳定，想切换到  unpkg.com 服务上来，但发现只有 V2 版本的地址，并未找到 V1 版本的引用地址，不知道哪位小伙伴知道可以分享下呢，谢谢。

**Comment by lizheming** [comment, marked as answer]
https://unpkg.com/@waline/client@v1

**Reply by elkan1788** [reply]
真没想到后面那个 W 是大写字母 :mute:

**Comment by elkan1788** [comment]
@lizheming  想再请教一下，在 vercel 中部署时，将版本调整为了1.6.0，如下：
``` javascript
{
  "name": "template",
  "version": "0.0.1",
  "private": true,
  "dependencies": {
    "@waline/vercel": "1.6.0"
  }
}
```
但发现其引用的是 `jsdelivr` CDN 服务，这个路径有没办法可以修改呢？ 谢谢。

**Reply by lizheming** [reply]
serverURL 首页那个无法修改，那个只是一个示例，不影响功能使用

**Reply by elkan1788** [reply]
但是影响到了管理界面的功能使用哦，发现引用的路径也是来自 `jsdelivr` CDN 服务😭

```
<script src="https://cdn.jsdelivr.net/npm/@waline/admin"></script>
```

**Reply by lizheming** [reply]
https://github.com/walinejs/waline/issues/776
